src/Raven.Server/Utils/MicrosoftLogging/WebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Raven.Server.Config.Categories;
+using Sparrow.Logging;
+
+namespace Raven.Server.Utils.MicrosoftLogging;
+
+public static class WebHostBuilderExtensions
+{
+    public static IWebHostBuilder ConfigureMicrosoftLogging(this IWebHostBuilder hostBuilder, ref LoggingSource loggingSource, LogsConfiguration configurationLogs)
+    {
+        if (configurationLogs.DisableMicrosoftLogs)
+            return hostBuilder;
+        
+        loggingSource = new LoggingSource(LogMode.Information, configurationLogs.MicrosoftLogsPath.FullPath, "Microsoft Log", TimeSpan.MaxValue, long.MaxValue);
```

Comment:
Add options with retention and size (nullable). If they are null, use the default ones

Reply:
I added all the log configurations